crates/bevy_light/src/probe.rs

```diff
@@ -96,14 +100,82 @@ pub struct EnvironmentMapLight {
     pub affects_lightmapped_mesh_diffuse: bool,
 }
 
+impl EnvironmentMapLight {
+    /// An environment map with a uniform color, useful for uniform ambient lighting.
+    pub fn solid_color(assets: &mut Assets<Image>, color: Color) -> Self {
```

Comment:
```suggestion
    pub fn solid_color(assets: &mut Assets<Image>, color: impl Into<Color>) -> Self {
```

so that e.g. `tailwind::AMBER_100` works :)